commands/list_endpoints.go

```diff
@@ -61,30 +61,59 @@ func endpointsTable(args listEndpointsArguments) string {
 			color.YellowString("gsctl login <email> -e <endpoint>"))
 	}
 
-	// table headers
-	output := []string{
-		strings.Join([]string{
-			color.CyanString("ENDPOINT URL"),
-			color.CyanString("EMAIL"),
-			color.CyanString("SELECTED"),
-			color.CyanString("LOGGED IN"),
-		}, "|"),
-	}
-
 	// get keys (URLs) and sort by them
 	endpointURLs := make([]string, 0, len(config.Config.Endpoints))
 	for u := range config.Config.Endpoints {
 		endpointURLs = append(endpointURLs, u)
 	}
 
+	// detect if we want to show the alias column
+	haveAlias := false
+	for _, endpoint := range endpointURLs {
+		if config.Config.Endpoints[endpoint].Alias != "" {
+			haveAlias = true
+		}
+	}
```

Comment:
A good practise is to have a dedicated function for such things. It is reusable and easer to use in the code.